pages/bare_metal_cloud/dedicated_servers/personal-templates-end-of-life/guide.en-gb.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Personal installation templates end of life
+excerpt: Information about removal of your personal installation templates & workaround
+updated: 2025-04-28
+---
+
+**This is an important piece of information, please read the entire message carefully.**
+
+## Context
+
+OVHcloud is constantly evolving and must adapt its offerings to the increasingly resilient and regulated cloud market. In order to meet those new challenges the **personal template for dedicated servers OS installation** will be fully decommissioned the **7th of October 2025**.
+
+This service was used to request an OS installation on a dedicated servers by targeting a personal template instead of an operating system with the following elements:
+- operating system to be installed
+- partitioning customization layout,
+- hardware RAID configuration,
+- hostname,
+- SSH public key,
+- post installation script.
+
+This decommissioning will be done in 2 steps:
+
+### Step 1: 17th of June 2025<a name="step-1"></a>
+
+- Customers will no longer be able to **edit** or **add** new personal templates
+- Customers having personal template(s) and active dedicated servers services will receive custom technical informations by email to help them migrate their personal template(s) using the new API call.
+
+### Step 2: 7th of October 2025<a name="step-2"></a>
+
+- Customers will no longer be able to access their personal templates
+- All personal templates data will be deleted
+
+Note that customers will still have the email from the 17th of June 2025 with technical documentation based on their personal templates data to migrate to new API call.
+
+## Instructions
+
+The following section contains specific information per usage of the service.
+
+### Using OVHcloud, SoYouStart or Kimsufi Control Panel<a name="ux"></a>
+
+The `Install one of your templates`{.action} wizard to install an operating system on a dedicated server has already been removed since mid-April. If you were using that wizard you have 3 long-term possibilities:
```

Comment:
```suggestion
The `Install one of your templates`{.action} wizard to install an operating system on a dedicated server has already been removed since mid-April 2025. If you were using that wizard you have 3 long-term possibilities:
```